indico/modules/events/papers/templates/reviewing/timeline.html

```diff
@@ -11,6 +11,11 @@
             {% else %}
                 <div class="paper-files-list flexrow f-a-baseline">
                     <div class="paper-files-list-label f-self-no-shrink">
+                        {% if paper.event.cfp.hide_accepted %}
+                            <span title="{% trans %}Accepted papers are currently hidden from participants{% endtrans %}"
+                                class="category-action left icon-eye-blocked active" style="padding-right:0.2rem;">
```

Comment:
```suggestion
                            <span title="{% trans %}Papers are currently hidden from other participants{% endtrans %}"
                                  class="category-action left icon-eye-blocked active"
                                  style="padding-right:0.2rem;">
```

Similar to the previous improvements, it's clearer that way.
Also fixed the indentation/wrapping.